README.md

```diff
@@ -6,6 +6,21 @@
 Usage:
   bundle exec rake
 
+  There are three environment variables that control how the tests are run.
+
+  TEST_REPO: This is the repository which will be tested.
+  CORE_REPO: If TEST_REPO is a plugin and you need to specify a different core branch
+  GEM_REPOS: Optional other plugins/gems which are needed to run the tests
+
+  All of these support the following formats:
+  Remote: [Org/]Repostory[@Ref][#PR]
```

Comment:
`[Org/]Repostory[@Ref|#PR]`  (Using the | to denote an "or")

Reply:
:+1: